go/src/meguca/db/bans.go

```diff
@@ -144,9 +144,21 @@ func Ban(board, reason, by string, expires time.Time, log bool, ids ...uint64) (
 
 	if !IsTest {
 		for _, post := range posts {
+			mLog, err := GetPostModLog(post.id)
+
+			if err != nil {
+				return err
+			}
+
+			err = auth.ModLogPost(post.id, post.op, mLog)
```

Comment:
What is this for?

Reply:
https://github.com/bakape/meguca/blob/master/go/src/meguca/db/admin.go#L77-L92
https://github.com/bakape/meguca/blob/master/go/src/meguca/websockets/feeds/feed.go#L21
https://github.com/bakape/meguca/blob/master/go/src/meguca/websockets/feeds/feed.go#L311-L313
https://github.com/bakape/meguca/blob/master/go/src/meguca/websockets/feeds/feeds.go#L27
https://github.com/bakape/meguca/blob/master/go/src/meguca/websockets/feeds/feeds.go#L303-L318
It's the whole reason this works.

Reply:
Too roundabout and leads to messier and harder to follow code. Could you centralise writing mod log to posts by moving it to `logModderation()`?

Reply:
We're not writing mod-log to posts server-side, we are sending mod-log info pertaining to a post to the client via websockets.
What I can do is change auth.ModLogPost to common.ModLogPost, remove the mLog input, and have it fetch from the db from websockets instead.